.gitpod.Dockerfile

```diff
@@ -16,7 +16,8 @@ RUN sudo apt-get update \
     sox gfortran \
  && sudo rm -rf /var/lib/apt/lists/*
 
-RUN cd ~/GitHub/r-with-intel-mkl/ \
+RUN mkdir ~/r-with-intel-mkl \
+ && cd ~/r-with-intel-mkl/ \
  && wget -O- https://apt.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS-2019.PUB | sudo apt-key add - \
  && sudo sh -c 'echo deb https://apt.repos.intel.com/mkl all main > /etc/apt/sources.list.d/intel-mkl.list' \
- && sudo apt-get update && sudo apt-get install intel-mkl-64bit
+ && sudo apt-get update && sudo apt-get install -y intel-mkl-64bit-2020.4-912
```

Comment:
why specific version?

Reply:
Because I use `intel-mkl-64bit` is a virtual package provided.
Need to manually select a package.

```
sudo apt-get install intel-mkl-64bit
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Package intel-mkl-64bit is a virtual package provided by:
  intel-mkl-64bit-2020.4-912 2020.4-912
  intel-mkl-64bit-2020.3-111 2020.3-111
  intel-mkl-64bit-2020.2-108 2020.2-108
  intel-mkl-64bit-2020.1-102 2020.1-102
  intel-mkl-64bit-2020.0-088 2020.0-088
  intel-mkl-64bit-2019.5-075 2019.5-075
  intel-mkl-64bit-2019.4-070 2019.4-070
  intel-mkl-64bit-2019.3-062 2019.3-062
  intel-mkl-64bit-2019.2-057 2019.2-057
  intel-mkl-64bit-2019.1-053 2019.1-053
  intel-mkl-64bit-2019.0-045 2019.0-045
  intel-mkl-64bit-2018.4-057 2018.4-057
  intel-mkl-64bit-2018.3-051 2018.3-051
  intel-mkl-64bit-2018.2-046 2018.2-046
  intel-mkl-64bit-2018.1-038 2018.1-038
  intel-mkl-64bit-2018.0-033 2018.0-033
You should explicitly select one to install.

E: Package 'intel-mkl-64bit' has no installation candidate
```

Reply:
This is the intel doc for install [Intel® oneAPI Math Kernel Library (oneMKL)](https://www.intel.com/content/www/us/en/developer/articles/guide/installing-free-libraries-and-python-apt-repo.html)

Reply:
I think I can use `kaldi/tools/extras/install mkl.sh` to install it.